From e56f92d5b197d0be3ddea654ef9a2afc06871ccb Mon Sep 17 00:00:00 2001
From: Andrew Bayer <andrew@cloudera.com>
Date: Thu, 22 Dec 2011 10:58:29 -0800
Subject: [PATCH 028/795] MR1: CLOUDERA-BUILD. Adding cloudera-snapshots to ivy resolvers.
 (cherry picked from commit 49e48f1c07497bc3a40c045c89b89d73a94656b1)

(cherry picked from commit a7078ecdb77707d7449670ca27340f6a08d57649)
---
 ivy/ivysettings.xml |   15 ++++++++++++++-
 1 files changed, 14 insertions(+), 1 deletions(-)

diff --git a/ivy/ivysettings.xml b/ivy/ivysettings.xml
index e8d6625..de98674 100644
--- a/ivy/ivysettings.xml
+++ b/ivy/ivysettings.xml
@@ -45,6 +45,9 @@
   <property name="ext.releases"
             value="http://repository.cloudera.com/artifactory/ext-release-local/"
             override="false" />
+  <property name="snapshots.cloudera.com"
+            value="https://repository.cloudera.com/artifactory/libs-snapshot-local/"
+            override="false"/>
 
   <!-- pull in the local repository -->
   <include url="${ivy.default.conf.dir}/ivyconf-local.xml"/>
@@ -74,10 +77,19 @@
       pattern="${maven2.pattern.ext}"
       m2compatible="true"
       />
-    <chain name="default" dual="true">
+    <ibiblio name="cloudera-snapshot"
+             root="${snapshots.cloudera.com}"
+             m2compatible="true"
+             checkmodified="true"
+             changingPattern=".*SNAPSHOT"/>
+
+    <chain name="default" dual="true"
+           checkmodified="true"
+           changingPattern=".*SNAPSHOT">
       <resolver ref="local"/>
       <resolver ref="maven2"/>
       <resolver ref="cdh-releases"/>
+      <resolver ref="cloudera-snapshot"/> 
     </chain>
     <chain name="internal">
       <resolver ref="local"/>
@@ -90,6 +102,7 @@
       <resolver ref="maven2"/>
       <resolver ref="cdh-releases"/>
       <resolver ref="apache-snapshot"/>
+      <resolver ref="cloudera-snapshot"/> 
     </chain>
   </resolvers>
   <modules>
-- 
1.7.0.4

